backend/tracim_backend/command/search.py

```diff
@@ -134,4 +134,4 @@ def take_app_action(self, parsed_args: argparse.Namespace, app_context: AppEnvir
         )
         print("delete index")
         self.search_api.delete_index()
```

Comment:
```suggestion
        self.search_api.delete_index()
```
Plural form may be better now, we do delete multiples index, no ?